src/Bladeburner/Bladeburner.ts

```diff
@@ -645,8 +645,8 @@ export class Bladeburner {
       }
     } else if (chance <= 0.7) {
       // Synthoid Riots (+chaos), 20%
-      sourceCity.chaos += 1;
-      sourceCity.chaos *= 1 + getRandomIntInclusive(5, 20) / 100;
+      sourceCity.changeChaosByCount(1);
+      sourceCity.changeChaosByPercentage(getRandomIntInclusive(5, 20));
```

Comment:
Similar to the diplomacy change, `changeChaosByPercentage` wants percentage points, so we can just take the `getRandomIntInclusive` directly.